plugins/in_tail/tail.c

```diff
@@ -680,6 +680,13 @@ static struct flb_config_map config_map[] = {
      "Allows to skip empty lines."
     },
 
+#ifdef __linux__
```

Comment:
Usually system guards use the definitions from the CMakeLists: https://github.com/fluent/fluent-bit/blob/9652b0dc6fd9e59ef87c3bdc59c89da9abed995b/CMakeLists.txt#L30

(Funnily enough this would be the first time `FLB_SYSTEM_LINUX` is used :smile:)

Reply:
I'm a bit confused, as there are many other parts in the code where __linux__ is used, like: https://github.com/fluent/fluent-bit/blob/9652b0dc6fd9e59ef87c3bdc59c89da9abed995b/src/flb_utils.c#L1361
or
https://github.com/fluent/fluent-bit/blob/9652b0dc6fd9e59ef87c3bdc59c89da9abed995b/plugins/in_node_exporter_metrics/ne_cpu.h#L27

Reply:
Interesting. I don't know why that's the case. There's lots of cases where `FLB_SYSTEM_WINDOWS` and `FLB_SYSTEM_MACOS` are used as system guards, I don't know why that isn't the case for Linux. Perhaps it's fine to leave this then.

Reply:
Yeah, it would be better to use it. However, I sometimes use `__linux__` guards. So, it's OK for me.